src/actions/articleActions.js

```diff
@@ -113,3 +117,28 @@ export const likeDislike = (payload, slug) => dispatch => {
       return toast.error(error.response.data.detail, { autoClose: false, hideProgressBar: true });
     });
 };
+
+export const updateArticle = (slug, newData) => dispatch => {
+  toast.dismiss();
+  dispatch(editArticleInititated(true));
+  return axiosInstance
+    .put(`/api/articles/${slug}/`, newData)
+    .then(response => {
+      dispatch(editArticleSuccess(true));
+      toast.success(
+        response.data.message,
+        { autoClose: 3500, hideProgressBar: true },
+      );
+    })
+    .catch((error) => {
+      let errorMessage = '';
+      if (error.response.status === 403) {
+        errorMessage = 'Re-login and try again';
+      }
+      if (error.response.status === 404) {
+        errorMessage = 'Please enter valid text';
```

Comment:
Does `valid text` help the user determine whether its the `title, body or description` with the issue? Is there an error message that returns with this error response?

Reply:
The front-end already caters for validation of `title` and `description` fields. I have updated the message to point to the article body specifically.